db/knex_migrations/2023-11-28-2022-add-zookeeper-column.js

```diff
@@ -0,0 +1,18 @@
+exports.up = function (knex) {
+    // update monitor.push_token to 32 length
+    return knex.schema.alterTable("monitor", function (table) {
+        table.string("zookeeper_host", 255);
```

Comment:
Please use the `hostname` and `timeout` fields instead.
I don't see a reason why we shoud add these fields ^^